corehq/apps/userreports/tests/test_report_filters.py

```diff
@@ -784,6 +807,75 @@ def test_filter_value(self):
         }
         filter_value = LocationDrilldownFilterValue(filter, ['Middlesex'])
         self.assertDictEqual(filter_value.to_sql_values(), {'block_id_drill_0': 'Middlesex'})
+        self.assertEqual(
+            str(filter_value.to_sql_filter().build_expression(mock_table)),
+            'block_id IN (:block_id_drill_0)'
+        )
+        self.assertEqual(
+            filter_value.to_sql_values(),
+            {'block_id_drill_0': 'Middlesex'}
+        )
+
+    def test_ancestor_expression(self):
```

Comment:
Could be renamed to `test_ancestor_expression_missing_location_type`

Reply:
Sure